app/assets/javascripts/admin/addon/controllers/admin-badges/show.js

```diff
@@ -1,35 +1,28 @@
 import { tracked } from "@glimmer/tracking";
```

Comment:
the biggest change here is to have stopped using `bufferedProperty` given form-kit includes its own proxy